### PR TITLE
Add agent files

### PR DESCRIPTION
**Pull request by bugblasterX**
Add changes to docker compose, Dockerfile and additional confuguration to run agent with slurm

### CODE REVIEW

File: Dockerfile.ctld [hunk 1]
```diff
@@ -0,0 +1,106 @@
+FROM rockylinux:8
```
Comment (livenson) [inline Dockerfile.ctld:1]: Why do you need to copy the full file? why not to simply extend existing Dockerfile?

File: docker-compose.yml [hunk 2]
```diff
@@ -49,6 +54,8 @@ services:
       - "slurmdbd"
     networks:
       - slurm-network
+    environment:
+      - REQUESTS_VERIFY_SSL=false
```
Comment (livenson) [inline docker-compose.yml]: let's not set it here 
if needed for some test, better to set within a single test

File: supervisord.conf [hunk 3]
```diff
@@ -0,0 +1,33 @@
+[supervisord]
+nodaemon=true
+user=root
+
+
+[program:entrypointscript]
+#should  run after slurmdbd started
```
Comment (livenson) [inline supervisord.conf]: extra space

File: supervisord.conf [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+command=/usr/local/bin/docker-entrypoint.sh slurmctld
+autorestart=false
+startretries = 1
```
Comment (livenson) [inline supervisord.conf]: here and in the rest of file - spacing around = is inconsistent

File: waldur-site-agent-config.yaml [hunk 5]
```diff
@@ -0,0 +1,37 @@
+sentry_dsn: "" # Data Source Name for Sentry (more info https://docs.sentry.io/product/sentry-basics/dsn-explainer/).
```
Comment (livenson) [inline waldur-site-agent-config.yaml:1]: is this default config file? I think then better to take it from upstream / develop branch

File: Dockerfile.ctld (hunk 1 above)
Comment (bugblasterX) [reply Dockerfile.ctld:1]: I believe the current dockerfile is used in the other containers, and we don't want the agent's setup there? Also some python dependencies and pip version were incompatible with our agent. 
Do you mean to use only 1 docker file?

**Pull request merged**